src/routes/oauth2/middleware.js

```diff
@@ -29,10 +30,16 @@ module.exports = {
     const requestJWT = req.jwt;
     try {
       if (requestJWT) {
-        const apiResponse = await req.axios.post(sessionPath, {
-          request: req.jwt,
-          client_id: req.session.authParams.client_id,
-        });
+        const userIp = userIpAddress(req);
+
+        const apiResponse = await req.axios.post(
+          sessionPath,
+          {
+            request: req.jwt,
+            client_id: req.session.authParams.client_id,
+          },
+          { headers: { "x-forwarded-for": userIp } }
```

Comment:
This is what we've been asked to do - and I was the one who narrowed the focus :)

I'd be interested in investigating how this could work as a default for all requests - similar to how the scenario header is added in debug mode - https://github.com/alphagov/di-ipv-cri-common-express/blob/main/src/lib/axios.js#L24

Reply:
Based on a number of separate conversions, we should move this feature into the common headers as linked above.

This removes the need for each CRI to implement this feature, which avoid a lot of unnecessary copying and pasting of code.